WordPress/Classes/Services/BlogService.m

```diff
@@ -108,12 +124,15 @@ - (Blog *)lastUsedBlog
         return nil;
     }
 
-    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Blog"];
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"visible = YES AND url = %@", url];
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:BlogEntity];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%@ AND url = %@", @"visible = YES", url];
```

Comment:
Just to add to my previous comment, this is actually bad (regardless of style) because the predicate parser fails:

```2015-03-10 13:51:19.923 WordPress[1675:814792] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Unable to parse the format string "%@ AND url = %@"'```

So rolling this change back, as we discussed, should fix it.